docs/templates/includes/theme-variables-row.md.njk

```diff
@@ -1,8 +1,17 @@
 <tr>
     <td>${{ variable.context.name }}</td>
     <td>{{ variable.type }}</td>
-    <td>{% if variable.context.value | is_color === true %}<span class="color-preview" style="background-color: {{variable.context.value}}"></span>{% endif %}{{ variable.context.value }}</td>
-    <td>{% if variable.resolvedValue | is_color === true %}<span class="color-preview" style="background-color: {{variable.resolvedValue}}"></span>{% endif %}{{ variable.resolvedValue }}</td>
+<td>
```

Comment:
May you trim the lines as before, e.g.:
```
<td>{% if variable.context.value | is_color === true %}<span class="color-preview" style="background-color: {{variable.context.value}}"></span>{% endif %}{% if variable.context.value %}`{{ variable.context.value }}`{% endif %}</td>
<td>{% if variable.resolvedValue | is_color === true %}<span class="color-preview" style="background-color: {{variable.resolvedValue}}"</span>{% endif %}{% if variable.resolvedValue %}`{{ variable.resolvedValue }}`{% endif %}</td>
```

So, the generated output doeesn't include so many new lines and produce for example  <td>`1px`</td>

Reply:
This is still a `md` file so the new lines won't appear in the final HTML - their only purpose is to trigger code-snippet parsing, so the final formatting will be correct, without new lines.